### DISCUSSION TITLE
Pro User Here- Sever lag issue during login

### DISCUSSION THREAD

**Discussion by JamesTuttleTrazi** [rustdesk/rustdesk#7607] [Q&A] [answered]
Hello! We setup a free Rust relay server on AWS and used it for a month to test out RDP with our 15-20 RustDesk clients. Login was almost immediate through our server, and the experience was so good that we purchased a Pro license. Our DevOps team setup the Pro server just like the free one (AWS Docker), with the same key and port settings. 

Logging in now is REALLY slow, and can take up to 30 seconds for connection. Once it's connected the speed is fast.  

What things should we check to find the issue? We know it's not a bandwidth or server resource issue (4 CPU 4GB Ram reserved).

Thanks so much!!

**Comment by rustdesk** [comment]
We exactly had a user have the same issue while switching from oss to pro.


>> It seems something wrong with your key file (not a valid secret key may cause this) while migrating.

> You were right, it was a missing key. It was present as a file but apparently not in the webinterface. After manually adding the secret key, it now works.

**Comment by JamesTuttleTrazi** [comment]
Thank you!

We have double checked and verified that our key's are correct.

Here is the full login details for our server. Could you please test and
send another solution? IT is hosted in a Docker on AWS, so we know speed

....

**Reply by rustdesk** [reply]
I just tested, I can not find any issue. I can connect within 4 seconds, considering I locate in Asia, the speed is fine.

**Comment by JamesTuttleTrazi** [comment]
We are having this issue when using our Rust Desk Pro internal server. I
can still connect unattended to our machines that were setup 6 months ago,
but now all machines that we connect through our relay server prompt for
connect authorization.

I have spent the last hour reviewing all settings in both the rust client
and rust server. There is no setting that is requiring "Please wait for the
remote side to accept your request"- and since our original setups DO allow
unattended access, this must be a software bug???

This is creating a HUGE problem for us, and the reason we purchased Rust
Desk Pro- Thanks in advance for any assistance!!

**Reply by rustdesk** [reply]
Could you show me this picture on controlled side? You might modify strategy on web console by mistake.

![image](https://github.com/rustdesk/rustdesk/assets/71636191/fa64d06d-b0bb-457a-a613-65a69960b22d)

**Comment by JamesTuttleTrazi** [comment]
Good morning! Here is a screen shot from the client computer that we are
trying to connect to:

[image: ClientSide.png]

We only use our Rust Pro Relay server for internal machines, so the
security is open. We need all our clients to allow unattended login access.

Also just FYI we keep our Rust clients on the latest software version. The
older clients which were setup 5 months ago still allow unattended login-
we are only having this problem with new computer setups.

Thanks so much for your help!!!

**Reply by rustdesk** [reply]
I can not see your picture, but I guess you have not set permanent password on the computers having problem.

**Comment by JamesTuttleTrazi** [comment]
Please see the attachments- I have included both a PNG and a PDF of the
same screen, for which one is easier for you to open.

We have the same permanent password set on all our clients, to make access
easy. It is also set on the two new units, but they still prompt for login
every time.

**Reply by rustdesk** [reply]
Bro, you replied to github, it can not display your attachments.

**Comment by rustdesk** [comment]
@fufesou  follow it up.

**Comment by fufesou** [comment]
> We have the same permanent password set on all our clients, to make access
easy. It is also set on the two new units, but they still prompt for login
every time.

@JamesTuttleTrazi Hi

Can you login with the preset password? Have you enabled the remember option?

![image](https://github.com/rustdesk/rustdesk/assets/13586388/27b9aa63-2b07-4efb-9b3b-40d28cd3aecb)

**Comment by JamesTuttleTrazi** [comment]
<img width="548" alt="ClientSide" src="https://github.com/rustdesk/rustdesk/assets/131198091/950844e5-dc18-46ab-a111-722483e0c993">

**Reply by JamesTuttleTrazi** [reply]
Sorry about that! I thought I was just emailing a support account LOL

**Comment by JamesTuttleTrazi** [comment]
I will test the save password login option now and see if that resolves the issue

**Comment by JamesTuttleTrazi** [comment, marked as answer]
Just FYI, we found the problem! Placing a check in the "Enable blocking user input"  box now allows unattended login. I misunderstood that meaning of that box.

Thanks so much for your assistance!!

**Reply by fufesou** [reply]
![image](https://github.com/rustdesk/rustdesk/assets/13586388/41197fa7-9f80-4eaa-b658-51d2f160c730)

Do you mean this option?

The weird thing is, I can use unattended login with or without this option set.

**Comment by JamesTuttleTrazi** [comment]
That is weird- does anyone know what exactly this option is for, and what's
supposed to change when you check it or un-check it?

I don't honestly understand what the problem was or why adding a check mark
to this box resolved our issue, but we are not able to login unattended to
all three computers that were having the problem. I changed nothing else,
except for placing a check in that box, so it clearly fixed our problem.

**Reply by fufesou** [reply]
> what's supposed to change when you check it or un-check it?

This option is used to block the other users' input when the controlled side is Windows.

<img width="243" alt="1719364557306" src="https://github.com/rustdesk/rustdesk/assets/13586388/58052caa-a22f-4e1f-b1af-5f96e2d8f0f0">

If it is checked, the other users or other connections cannot input.

**Comment by JamesTuttleTrazi** [comment]
Thanks so much!!  This issue has been resolved for all our users.
